### DISCUSSION TITLE
Listening for "find in document" button event

### DISCUSSION THREAD

**Discussion by rmdorsey** [stephanrauh/ngx-extended-pdf-viewer#2450] [Q&A]
Is there an event to listen for when the user clicks to open/close the "find in document" button? 

![image](https://github.com/user-attachments/assets/5a07b2b2-69b0-4d81-a204-45c99b69f0d4)

thanks!

**Comment by stephanrauh** [comment]
Yes. You're looking for `[(findbarVisible)]`.

**Reply by rmdorsey** [reply]
thanks!  I found it here, but it didn't specify the event function to listen for.  I tried (findbarVisibleChange) and it worked!  Thanks!

![image](https://github.com/user-attachments/assets/0d10f31a-ead6-4d47-8bbd-d9dac830700d)
